### PR TITLE
[FIX] website_slides: fix various UI issues with comments/likes

### LINKED ISSUE #29901: [12] website: Comments are not displayed on Slides
Impacted versions: 12.0

Reproduced on latest Runbot v12 (http://421723-12-0-b969c0.runbot13.odoo.com/slides/public-channel-1) w/o modifying any views

Steps to reproduce:
screenshot

Current behavior:
dd

Expected behavior:
screenshot

Video/Screenshot link (optional):

====================================================================
Ps: Adding small UI issue on same page
screenshot

Current behavior. 
screenshot

Expected behavior:
screenshot

@JKE-be  You can close this right away if you find this near to expected behavior in current framework.

### PR DESCRIPTION
**Pull request by awa-odoo**
Task #1923443

Purpose
=======

This commit fixes several problems with the website UI of the slides module:

1. Probably due to the migration to bootstrap 4, the active tab panel in the 'slide_detail_view' was not displayed on page loading (and only when switching back and forth in the tabs).

2. When adding a like/dislike to a slide, the like/dislike counter was not properly updated.

closes #29901

### CODE REVIEW

File: addons/website_slides/views/website_slides.xml [hunk 1]
```diff
@@ -462,7 +462,7 @@
                                             tabindex="0"
                                             data-toggle="popover">
                                             <i class="fa fa-thumbs-up fa-1x" role="img" aria-label="Likes" title="Likes"/>
-                                            <t t-esc="slide.likes"/>
+                                            <span class="o_wslides_like_dislike_count" t-esc="slide.likes"/>
```
Comment (tde-banana-odoo) [inline addons/website_slides/views/website_slides.xml]: wslides ?

Comment (awa-odoo) [reply addons/website_slides/views/website_slides.xml]: According to my secret informant in the website office, the new name convention for classes is something like:
- o (for le Odoo)
- w (for le website)
- slides (for le module)

He gave me the following example: o_wevent_registration_list

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by tde-banana-odoo** [comment]
@robodoo r+ retry redo do stufff please I am alone in the drak don't try to ff vive les frites

**Comment by tde-banana-odoo** [comment]
@robodoo retry

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by awa-odoo** [comment]
@robodoo retry

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**